### DISCUSSION TITLE
Supabase SMTP / Emails not working?

### DISCUSSION THREAD

**Discussion by AdrianArtiles** [supabase/supabase#1236] [Questions] [answered]
I was experiencing no emails being sent (password reset, confirmation emails, etc.) when it seemed to be working a few hours prior.

I finally switched over to a custom SMTP server with credentials, and emails seemed to be working again.

I checked http://status.supabase.io but did not see anything mentioning the issue I experienced.

Was there an SMTP issue? Or could there have been an issue / setting / etc. more unique to me that I could be missing?

Thanks in advance for any light that could be shed!

**Comment by silentworks** [comment]
Did you check your spam folder, maybe the emails from the supabase smtp server went there instead. I was using the supabase SMTP server yesterday to send email and I got it. I don't think there is anything in the settings that could have gone wrong especially since when you used the custom SMTP server it worked without a issue.

**Reply by AdrianArtiles** [reply]
Thanks for taking the time to respond Andrew!
When I was experiencing the issue I had the same idea and checked the spam folder but did not see any emails caught there.

I tested sending emails again this morning and am still not getting emails.
I then tested again with email addresses at different domains and it seems some of them are getting through.

Emails to gmail.com and icloud.com have gotten through, but emails to outlook.com and a custom domain have not (and do not appear in those spam folders either).

It seems there's some sort of filtering happening at an email provider level? Not sure. Could somehow still be unique to me, but not sure in what way. I'm sending emails with the original templates from a brand new supabase account, so it should all be out-of-the-box stuff. Maybe some email providers don't like that the links are to localhost and filter them out? Seems unlikely, but not sure of a more likely explanation at the moment.

**Comment by AdrianArtiles** [comment, marked as answer]
(copy & pasting some of my reply here to mark as a possible answer)

It seems some emails are filtered out entirely (not even appearing in the spam folder) from the Supabase provided SMTP server.

In my personal testing emails to gmail.com and icloud.com have gotten through, but emails to outlook.com and a custom domain have not (and do not appear in those spam folders either).

It seems there's some sort of filtering happening at an email provider level? Not sure. Could somehow still be unique to me, but not sure in what way. I'm sending emails with the original templates from a brand new supabase account, so it should all be out-of-the-box stuff. Maybe some email providers don't like that the links are to localhost and filter them out? Seems unlikely, but not sure of a more likely explanation at the moment.

**Reply by pivot-dennis** [reply]
Hello, I am completely at a loss and can't find a satisfying solution at all.

Originally, I used the Supabase email service, but during testing, I accidentally triggered several bounce-backs and got banned from the internal Supabase email service. As a result, I had to switch to my custom SMTP.

At first, I configured the custom SMTP, and everything worked perfectly. I could register easily, received the confirmation emails—everything was great. Then, I wanted to switch to my production email, and it didn’t work there.

At first, I thought my SMTP configuration was wrong or that I had set up incorrect login credentials, but that wasn’t the case. Afterward, I switched back to the previously working test email: it also stopped working.
I didn’t change anything in my code, double-checked all login credentials multiple times—nothing works! A few hours later, it suddenly worked again without me changing anything. It worked for a short time, then stopped working again.

This behavior is catastrophic. Now I’m not even getting my error logs anymore when I try to create an account with an already existing email. This used to work flawlessly! I still haven’t changed anything in my code.

Has anyone had the same experiences or found a solution?

I’m desperate!

**Reply by stastix** [reply]
I'm still facing the same issue  
any fix ?
